src/applications/personalization/profile360/sass/profile-360.scss

```diff
@@ -70,3 +70,12 @@ input.usa-only-phone {
     margin: 0;
   }
 }
+form .payment-info-submit-button {
```

Comment:
we can't just target using `.payment-info-submit-button` since the CSS selector specificity we are trying to override is greater than `0 0 1 0`. So we can either add `form` like this or add `[type="submit"]` to the selector to match the `0 0 1 1` specificity we need to override.